examples/getting-started/Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM golang:1.9.4-alpine3.7
-
+FROM golang:1.9.4-alpine3.7 as builder
 WORKDIR /go/src/github.com/GoogleCloudPlatform/skaffold/examples/getting-started
-CMD ["./app"]
 COPY main.go .
-RUN go build -o app main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
```

Comment:
Do we really need `CGO_ENABLED=0 GOOS=linux`?

Reply:
No, removed it.